js/ui/html_editor/ui.html_editor.js

```diff
@@ -146,13 +153,27 @@ const HtmlEditor = Editor.inherit({
             transclude
         });
 
+        this._renderSubmitElement();
         this.callBase();
 
         this._updateContainerMarkup();
     },
 
-    _hasTranscludedContent: function() {
-        return this._$templateResult && this._$templateResult.length;
+    _renderSubmitElement: function() {
+        this._$submitElement = $("<textarea>")
+            .addClass(HTML_EDITOR_SUBMIT_ELEMENT_CLASS)
+            .attr("hidden", true)
+            .appendTo(this.$element());
+
+        this._setSubmitValue(this.option("value"));
+    },
+
+    _setSubmitValue: function(value) {
+        this._$submitElement.val(value);
+    },
+
+    _getSubmitElement: function() {
```

Comment:
This method never uses.

Reply:
It's redefine of the Editor's [method](https://github.com/DevExpress/DevExtreme/blob/18_2/js/ui/editor/editor.js#L282)